.github/actions/install-dpcpp/action.yml

```
@@ -19,6 +19,7 @@ runs:
       if: inputs.DPCPP_RELEASE == 'NIGHTLY'
       shell: bash
       run: |
+        sudo apt install -y libhwloc15
```

We can do something like this to install the package conditionally. This will avoid failures if `sudo` is not available: 
```suggestion
        dpkg -l | grep libhwloc15 > /dev/null || sudo apt install -y libhwloc15
```